packages/amplify-e2e-core/src/nexpect-reporter.js

```diff
@@ -3,6 +3,17 @@ const path = require('path');
 const { v4: uuid } = require('uuid');
 const localTemplatePath = path.resolve(__dirname, '../dist/index.html');
 
+const getShortNameForTestSuite = (
```

Comment:
tests file names have a structure like:
"C:\\<path>\\<to>\\mytest.test.ts" on windows
or
"home//<path>//<to>//mytest.test.ts" on linux